replit_river/session.py

```diff
@@ -267,18 +264,25 @@ async def _heartbeat(
                     ACK_BIT,
                 )
                 self._heartbeat_misses += 1
+                logging.debug(
+                    "heartbeat right now : {} {} {}".format(
+                        self._heartbeat_misses,
+                        self._transport_options.heartbeats_until_dead,
+                        self._transport_options.heartbeat_ms,
+                    )
+                )
                 if (
                     self._heartbeat_misses
-                    >= self._transport_options.heartbeats_until_dead
+                    > self._transport_options.heartbeats_until_dead
```

Comment:
during the initial code review i almost commented on this name because it's slightly unclear if this test should be `>` or `>=` and it looks like we tripped over it, heh. `max_failed_heartbeats` or `failed_heartbeat_limit` are probably a bit more common as names. Totally optional, just a thought for future changes.

Reply:
Added a TODO to do that, right now i prefer to keep it consistent with the river protocol to prevent confusion.